README.md

```diff
@@ -15,3 +15,7 @@
 
 ### HOW TO RUN ###
 ```foreman run python run.py```
+
```

Comment:
not sure what you were trying to accomplish here....? could you please make sure your code is more precise next time